src/index.ts

```diff
@@ -83,7 +83,7 @@ export function vega(vgView: VgView, options: Option = {}) {
 * It can also provide custom title and format for fields
 * options can be supplemented by vlSpec
 */
-export function vegaLite(vgView: VgView, vlSpec: Spec, options: Option = {}) {
+export function vegaLite(vgView: VgView, vlSpec: TopLevelExtendedSpec, options: Option = {}) {
```

Comment:
So should I change it back to `Spec`? But that's gonna cause an error because of mismatch type.